### DISCUSSION TITLE
Is there a way for several models to use same prompt?

### DISCUSSION THREAD

**Discussion by VitaliyS** [Comfy-Org/ComfyUI#745] [Q&A]
I want to generate several images with different models, but with the same prompt. How can I do it?

**Comment by ltdrdata** [comment]
Change text widget to input. And connect primitive node. Then, you can share same prompt.

**Comment by Technologicat** [comment]
Thank you!

I ended up here from a search engine, when trying to feed the same prompt into _Chroma_ and _Illustrious_, to use the latter as an img2img refiner in a combined workflow. These each use a completely different type of text encoder (_Chroma_ uses T5XXL, whereas _Illustrious_ uses CLIP).

I tried to search the node types in the GUI for "_prompt_", "_text_", "_string_", and "_note_", but the word "_primitive_" didn't occur to me.
